bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ThingStatusDetail.java

```diff
@@ -9,13 +9,15 @@
 
 /**
  * {@link ThingStatusDetail} defines possible status details of a {@link ThingStatusInfo}.
- * 
- * @author Stefan Bußweiler - Initial contribution
+ *
+ * @author Stefan Bußweiler - Initial contribution, added new status details
  */
 public enum ThingStatusDetail {
     NONE(0),
     HANDLER_MISSING_ERROR(1),
+    HANDLER_REGISTERING_ERROR(9),
```

Comment:
Is the thing status detail persisted anywhere, so that we have to keep the old id as they are? Otherwise, I would prefer to re-assign the ints to count from 1 to 10...